lxd/api.go

```diff
@@ -106,7 +106,8 @@ func restServer(d *Daemon) *http.Server {
 		uiHandlerErrorUINotEnabled := http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
 			w.Header().Set("Content-Type", "text/html")
 			w.WriteHeader(http.StatusServiceUnavailable)
-			fmt.Fprint(w, errorMessage)
+			_, err := fmt.Fprint(w, errorMessage)
+			logger.Warn(fmt.Sprintf("Could not write error message to writer object: %v", err), logger.Ctx{"url": r.URL, "method": r.Method, "remote": r.RemoteAddr, "err": err})
```

Comment:
You don't need the `fmt.Sprintf` here to contain the error as it is included in the `logger.Ctx`.

Reply:
Suggest:
```go
logger.Warn("Could not write error message to writer object", logger.Ctx{"url": r.URL, "method": r.Method, "remote": r.RemoteAddr, "err": err})
```

Also since we were unable to write the error message to the response writer it might be a good idea to put that error message into the log context.